Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL maintainer="zoltan@integritee.network"
 # By default we warp the service
 ARG BINARY_FILE=integritee-service
 
-COPY bin/enclave.signed.so bin/end.rsa bin/end.fullchain /usr/local/bin/
+COPY bin/enclave.signed.so /usr/local/bin/
```

Comment:
Removed the dummy certificate files, since they were replaced by a self-generated certificate a while back.